src/app/stack/stack-analyses.service.ts

```diff
@@ -87,6 +133,13 @@ export class StackAnalysesService {
     return body as StackReportModel;
   }
 
+  private extractTokenData(res: Response) {
+    let body = res.json() || {};
+    body['statusCode'] = res.status;
+    body['statusText'] = res.statusText;
+    return body as TokenDetailModel;
+  }
```

Comment:
Make this as lambda as it doesn't seem to be used elsewhere.

Reply:
updated it